protocol/avdl/chat1/local.avdl

```diff
@@ -1137,4 +1137,267 @@ protocol local {
     array<RateLimit> rateLimits;
   }
   ClearBotCommandsLocalRes clearBotCommandsLocal();
+
+
+  record RateLimitRes {
```

Comment:
this file is pretty big. any chance these it's not too hard to move all of these new types to a new file called something like `cli.avdl`? they're only used in the cli, right?

Reply:
done! 👍